tests/comfy_cli/command/generate/test_json_errors.py

```
@@ -0,0 +1,346 @@
+    assert _sole_envelope(r)["error"]["code"] == "generate_target_required"
+
+
+@pytest.mark.parametrize("flag", ["-h", "--help"])
+def test_help_flags_are_not_treated_as_a_flag_shaped_target(runner, flag):
+    """The guard must not swallow the help short-circuit."""
+    r = runner.invoke(cli_app, ["generate", flag])
+    assert r.exit_code == 0
+    assert "comfy generate" in r.stdout
+
+
+def test_json_stream_target_required_envelope_is_the_terminal_line(runner):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)